server/src/main/java/org/apache/iotdb/db/sync/sender/manager/LocalSyncManager.java

```diff
@@ -46,8 +54,36 @@ public LocalSyncManager(DataRegion dataRegion, Pipe pipe) {
 
   /** tsfile */
   @Override
-  public void syncRealTimeDeletion(Deletion deletion) {
-    syncPipe.collectRealTimeDeletion(deletion, dataRegion.getStorageGroupName(), dataRegionId);
+  public void syncRealTimeDeletion(
+      Deletion deletion,
+      List<TsFileResource> unsealedResources,
+      List<TsFileResource> sealedResources) {
```

Comment:
To avoid affecting the deletion performance, we should return directly if syncDelOp=false.

Maybe add a getter in TsFilePipe to get syncDelOp.

Reply:
done.